FireboltNETSDK/Client/FireboltConnection.cs

```diff
@@ -149,8 +150,16 @@ public override string ServerVersion
             }
         }
 
+        /// <summary>
+        /// Gets the name of the database to which to connect.
+        /// </summary>
+        /// <returns>The name of the database server to which to connect.</returns>
```

Comment:
`database server` implies this is a url, I assume this is the name? Let's remove mention of server altogether to avoid confusion.